worldedit-core/src/main/java/com/sk89q/worldedit/extension/factory/parser/mask/RadiusMaskParser.java

```diff
@@ -0,0 +1,35 @@
+package com.sk89q.worldedit.extension.factory.parser.mask;
+
+import com.boydti.fawe.object.mask.RadiusMask;
+import com.sk89q.worldedit.WorldEdit;
+import com.sk89q.worldedit.extension.factory.parser.RichParser;
+import com.sk89q.worldedit.extension.input.InputParseException;
+import com.sk89q.worldedit.extension.input.ParserContext;
+import com.sk89q.worldedit.function.mask.Mask;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.stream.Stream;
+
+public class RadiusMaskParser extends RichParser<Mask> {
+
+    public RadiusMaskParser(WorldEdit worldEdit) {
+        super(worldEdit, "{");
+    }
+
+    @Override
+    protected Stream<String> getSuggestions(String argumentInput, int index) {
+        if(index == 0 || index == 1){
+            return suggestPositiveDoubles(argumentInput);
+        }
+        return Stream.empty();
+    }
+
+    @Override
+    protected Mask parseFromInput(@NotNull String[] arguments, ParserContext context) throws InputParseException {
+        if(arguments.length < 2) return null;
+        int min = Integer.parseInt(arguments[0]);
+        int max = Integer.parseInt(arguments[1]);
+        System.out.println("radius: " + min + " | " + max);
```

Comment:
Better shift this to a debug to keep it or remove it before merging.

Reply:
Slipped through, will remove